designcompose/src/main/java/com/android/designcompose/squoosh/SquooshRoot.kt

```diff
@@ -116,6 +116,7 @@ internal class SquooshAnimationRenderingInfo(
     val action: AnimatedAction?,
     val variant: VariantAnimationInfo?,
     var startTimeNanos: Long = 0L,
+    var delayTimeMs: Int = 0,
```

Comment:
I don't like that we're mixing units and that the time units are in a basic type (rather than a sum type for each unit)... but [Compose does the same thing](https://developer.android.com/reference/kotlin/androidx/compose/animation/core/TweenSpec#TweenSpec(kotlin.Int,kotlin.Int,androidx.compose.animation.core.Easing)), and they don't even include the unit in the name of the `delay` argument. Compose does seem consistent in using `whateverMillis` when they do include the unit in the argument name, so maybe we should do that instead of `Ms` (*cries in Rust std::time::Duration*, but only for a few millis, or ms...).

Reply:
Done